### DISCUSSION TITLE
Geospatial maps

### DISCUSSION THREAD

**Discussion by robertjasoncross** [carbon-design-system/carbon-charts#1356] [General]
Are there any examples of geospatial maps using carbon charts?

**Comment by theiliad** [comment]
Hi 👋🏻 

Here's pretty much what we have https://carbondesignsystem.com/data-visualization/complex-charts/#geospatial-charts

**Reply by robertjasoncross** [reply]
Thanks. I am looking for any code examples...basic is fine. The items I am finding from the carbon design site examples are just images.

**Reply by theiliad** [reply]
We don't really have code examples or components available for those use cases, just mapbox themes that you can find here https://carbondesignsystem.com/data-visualization/complex-charts/#mapbox

**Reply by robertjasoncross** [reply]
Ah, so it is not a Carbon Component. Confusing, since it is listed with all of the others that are actual components...my misunderstanding.

I will just stick to using Leaflet, Thanks for the clarification.

**Reply by theiliad** [reply]
Yes the site offers design and dev resources.

Most of the dev resources offer demos & links to storybook/sandbox.

These aren't yet available in a web component library
